spec/features/meters/3_user_creates_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+feature 'create a meter', %Q{
+  As an Authenticated User
+  I want to create a new meter
+  So I can select that meter for new dances.
+  } do
+  # Acceptance
+  # User cannot create new meters unless user is signed-in
+  # User must povide Name and Desription
+  # If user is not signed-in they cannot see the link to add new meters.
+
+  let!(:anne) { FactoryGirl.create(:anne) }
+  let!(:dance) { FactoryGirl.create(:dance) }
+  new_meter = { name: "1/2", description: "I have no idea" }
+
+  pending 'authenticated user successfully create meter
+    with valid information' do
+    sign_in anne
+    visit new_dance_path
+    click_link "New Meter"
+
+    expect(page).to have_current_path(new_meter_path)
+
+    fill_in 'Name', with: new_meter[:name].to_s
+    fill_in 'Description', with: new_meter[:description].to_s
+    click_button "Create Meter"
+    visit meters_path
+
+    expect(page).to have_content(new_meter[:name])
+  end
+
+  pending 'authenticated user fail to create meter
+    with invalid information' do
+    sign_in anne
+    visit new_dance_path
+    click_link "New Meter"
+    fill_in 'Name', with: ""
+    fill_in 'Description', with: ""
+    click_button "Create Meter"
+
+    expect(page).to have_content(
+      "Name can't be blank, Description can't be blank"
+    )
+  end
+
+  scenario 'fail to create a meter with unauthenticated user' do
+    visit new_dance_path
+
+    expect(page).to have_content(
+      "You need to sign in or sign up before continuing."
+    )
+    expect(page).to have_current_path(new_user_session_path)
+  end
+
+  pending 'meter form should open in a new window, close, and new meter
+  should be in the meter select box of the new dance form'
+  # expect the form to open in new window?
+  # expect the form to close and be able to select the new meter from the
+  # select box?
+end
```

Comment:
So check out this post:

http://stackoverflow.com/questions/7612038/with-capybara-how-do-i-switch-to-the-new-window-for-links-with-blank-targets

It may be worth upgraging to capybara ~ 2.3 if you really want to test that a new window is open. It may be worth it to investigate capybara's window management API to find out some more features here.